test/passing/tests/collections-janestreet.ml.ref

```diff
@@ -0,0 +1,43 @@
+let _ =
+  [ a; b (* foooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo *) ]
+;;
+
+let [ fooooooooooooooooooooooooooooooo
+    ; fooooooooooooooooooooooooooooooo
+    ; fooooooooooooooooooooooooooooooo (* before end of the list *)
+    ]
+  =
+  [ fooooooooooooooooooooooooooooooo
+  ; fooooooooooooooooooooooooooooooo
+  ; fooooooooooooooooooooooooooooooo
+    (* after all elements *)
+    (* after all elements as well *)
+  ]
+;;
+
+let [| fooooooooooooooooooooooooooooooo
+     ; fooooooooooooooooooooooooooooooo
+     ; fooooooooooooooooooooooooooooooo
+    (* before end of the list *)
```

Comment:
The indentation is not right in patterns.

Reply:
I reviewed Params.ml and I have no idea what is causing this, especially since all patterns and all expressions are handled the same way and the length of the opening symbol is accounted for.

Reply:
All weird indentations should be fixed now